resources/views/buyers/checkout_confirm.blade.php

```diff
@@ -0,0 +1,185 @@
+@extends('layouts.app')
+
+@section('title', 'Buyer: Checkout Confirm Page')
+<style>
+    label:not(.form-check-label) {
+        margin-top: 0.5rem;
+        margin-bottom: 0rem;
+    }
+
+    .card-body img {
+        width: 50px;
+        height: 50px;
+    }
+
+    .card {
+        width: 40%;
+    }
+
+    .card-footer,
+    .card-header {
+        background-color: #f8fafc !important;
+    }
+</style>
```

Comment:
Same here

Reply:
I put this in checkout.css with commit [72317eb](https://github.com/kredo-school/16th_rankshty/pull/45/commits/72317ebe156c5527809ac9d84121c000bbc9037d).